bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -167,7 +172,15 @@ public String getId() {
     @JsonIgnore
     @Nullable
     public String getRedirectUrl() {
-        return GuideUtils.getRedirectUrl(redirect, getPath());
+        return getContextPath() + GuideUtils.getRedirectUrl(redirect, getPath());
+    }
+
+    @JsonIgnore
+    public String getContextPath() {
```

Comment:
We should include this function in interface as well with default value as empty.

https://github.com/adobe/aem-core-forms-components/blob/master/bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/models/form/FormContainer.java

Reply:
@ujjwal5 , contextPath is not a form container property or behaviour. I think we can make it private method here. submit or redirect is closely tied with formcontainer. I don't think there is a need to surface this at interface level.

Reply:
@pankaj-parashar I had to add it in the interface or else the test case wouldn't work. Hope thats fine. 
https://github.com/adobe/aem-core-forms-components/pull/477/files#diff-9fde01b43f289e2c95c2058a4ec5a8b5cc5fa4efa7a084f1b48214d4d1e7ec00R321

Reply:
On a side note, do we need to provide this method in v1 FormContainerImpl as well ?
@pankaj-parashar @sakshi-arora1